CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.4.0]
+### Added
+- Allow to retain tags history in case of renamed packages ([#157](https://github.com/MetaMask/auto-changelog/pull/157))
+  - Added --version-before-package-rename and --tag-prefix-before-package-rename options to validate command.
+- Remove `as` assertions, or fix to mitigate the risk of silent failure ([#158](https://github.com/MetaMask/auto-changelog/pull/158))
```

Comment:
This looks like a change that was mostly developer-only and didn't have much impact on consumers. I believe we did tweak the error message produced when a release candidate version already has a Git tag, so minimally we could include that here:

``` suggestion
- Improve error message produced when Git tag for release candidate version already exists ([#158](https://github.com/MetaMask/auto-changelog/pull/158))
```

But we could also just remove this.

``` suggestion
```

Up to you.